src/types/chart/chart.ts

```diff
@@ -203,3 +203,8 @@ export type GenericDefinition<T extends ChartWithDataSetDefinition> = Partial<
 > & {
   dataSets?: Omit<T["dataSets"][number], "dataRange">[];
 };
+
+export interface ZoomConfiguration {
+  readonly enabled?: boolean;
+  readonly sliceable?: boolean;
```

Comment:
I'm not sure to understand what `sliceable` is. In which situation is a chart zoomable but not sliceable ?

Reply:
`enabled` means the button to show/hide the slicer is visible. This is something like "allow the user to interact with the chart".
`sliceable` means the slicer is visible.
`wheelable` means we can CTRL+scroll to zoom directly in the chart (only when enabled is true and the axis is a linear axis)

Reply:
Oh that what it was. Is there really a point to have both tho ? Do we have a need for a chart where we _could_ zoom but the slider is not visible until we click a button ? IMO the there should be a single boolean that show/hide the slider